src/main/java/seedu/address/model/person/ContactContainsKeywordsPredicate.java

```diff
@@ -17,7 +16,7 @@ public ContactContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Person person) {
         return this.getKeywords().stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getPhone().value, keyword));
+                .anyMatch(keyword -> person.getPhone().value.toLowerCase().contains(keyword.toLowerCase()));
     }
 
     @Override
```

Comment:
Can keep in view to possibly abstract the `test` method into the superclass FindCommand if we decide to extend the logic